scripts/query/query.ts

```diff
@@ -0,0 +1,35 @@
+import type { SnapManifest } from '@metamask/snaps-utils';
+import fs from 'fs';
+
+import { MANIFESTS_FILE_LOCATION, getManifests } from './get-manifests';
+
+/**
+ *  Main function.
+ */
+async function main() {
+  const fileExists = fs.existsSync(MANIFESTS_FILE_LOCATION);
+  const forceDownload = process.argv.includes('--force-download');
+
+  if (!fileExists || forceDownload) {
+    await getManifests();
+  }
+  const manifests: SnapManifest[] = JSON.parse(
+    fs.readFileSync(MANIFESTS_FILE_LOCATION, 'utf8'),
+  );
+
+  /**
+   * Write your custom code to query the `manifests` array for the data you are interested in.
+   */
+
```

Comment:
So the idea is that you have to modify this script to query for different data?

Reply:
Yes.
We do not know what the criteria would be for our next search. Its unknown and dynamic in nature.
The current code is the "groundwork" to help for any filter/search criteria that could come up.